packages/ui/components/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -360,6 +360,16 @@ describe('<lion-input-datepicker>', () => {
         expect(el.__calendarMaxDate.toString()).to.equal(new Date('2019/07/15').toString());
       });
 
+      it('should validate default value', async () => {
+        const myDate = new Date('30/01/2022');
```

Comment:
Can we use a more clear wrong value?

Reply:
@gerjanvangeest  sure, I can add few more tests. I just wanted to use the value from issue #1984